R/database.R

```diff
@@ -1594,8 +1594,10 @@ get_cn_states = function(regions_list,
 #' Get all segments for a single (or multiple) sample_id(s).
 #'
```

Comment:
The parameters for this function should be similar (or the same) as other functions that work this way. Right now, a user must supply 
this_sample_id="some_id" or 
multiple_samples = TRUE, sample_list = c("some_sample", "another_sample") 

Why? 
Wouldn't it make more sense to ask the user to supply either this_sample_id or  these_samples_metadata and let the function figure out if they want data from multiple samples based on how many samples are in these_samples_metadata?

Reply:
I agree. in This PR, I have not touched this function other than updating the already existing parameter descriptions, but they should be updated as you're suggesting, of course.